root/tree/cloning/CMakeLists.txt

```
@@ -173,3 +173,8 @@ ROOTTEST_ADD_TEST(index
 ROOTTEST_ADD_TEST(assertBranchCount
                   MACRO assertBranchCount.C
                   OUTREF references/assertBranchCount.ref)
+
+ROOTTEST_ADD_TEST(clone-friend-tear-down
+                  COPY_TO_BUILDDIR files/tr1.root
+                  COPY_TO_BUILDDIR files/tr2.root
+                  COMMAND ${ROOT_hadd_CMD} -f tr_merge.root tr1.root tr1.root)
```

```suggestion
                  COMMAND ${ROOT_hadd_CMD} -f tr_merge.root ${CMAKE_CURRENT_SOURCEDIR}/tr1.root ${CMAKE_CURRENT_SOURCEDIR}/tr1.root)
```
(or whatever that CMake var is spelled)

---

fair enough.  I changed it even-though the existing code was following the other test's pattern.